src/vs/workbench/contrib/remoteTunnel/electron-sandbox/remoteTunnel.contribution.ts

```diff
@@ -712,6 +716,39 @@ export class RemoteTunnelWorkbenchContribution extends Disposable implements IWo
 			}
 		}));
 
+		this._register(registerAction2(class extends Action2 {
+			constructor() {
+				super({
+					id: RemoteTunnelCommandIds.closeUnusedPorts,
+					title: RemoteTunnelCommandLabels.closeUnusedPorts,
+					category: REMOTE_TUNNEL_CATEGORY,
+					menu: []
```

Comment:
When the remote system is NOT linux or if `remote.autoForwardPortsSource` is set to `output`, then none of the ports have a running process. 

When the remote system is linux and the setting `remote.autoForwardPortsSource` is not set to `output`, I think this command makes sense. I would add the command to the command palette in this case.